services/dy-dash/cc-rabbit-0d/src/cc-rabbit-0d/cc-rabbit-0d.py

```diff
@@ -56,58 +56,24 @@
 }
 GRAPH_HEIGHT = 400
 
-def get_empty_input_graph(xLabel='x', yLabel='y'):
-    fig = go.Figure(data=[], layout={})
-
-    fig['layout']['xaxis'].update(
-        title=xLabel,
-        gridcolor=osparc_style['gridColor']
-    )
-    fig['layout']['yaxis'].update(
-        title=yLabel,
-        gridcolor=osparc_style['gridColor']
-    )
-    margin = 10
-    y_label_padding = 50
-    x_label_padding = 30
-    fig['layout']['margin'].update(
-        l=margin+y_label_padding,
-        r=margin,
-        b=margin+x_label_padding,
-        t=margin,
-    )
-
-    fig['layout'].update(
-        autosize=True,
-        height=GRAPH_HEIGHT,
-        showlegend=False,
-        plot_bgcolor=osparc_style['backgroundColor'],
-        paper_bgcolor=osparc_style['backgroundColor'],
-        font=dict(
-            color=osparc_style['color']
-        )
-    )
-    return fig
-
-def get_empty_cols_graphs(labelPairs=[['x', 'y']]):
-    fig = tools.make_subplots(rows=1,
-                            cols=len(labelPairs),
-                            shared_xaxes=True,
-                            vertical_spacing=0.05
-    )
-
-    for idx, labelPair in enumerate(labelPairs):
+def give_fig_osparc_style(fig, xLabels=['x'], yLabels=['y']):
```

Comment:
Be careful with using mutables as defaults.Read  https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments